weaviate/collections/classes/cluster.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import dataclass
+from typing import Generic, List, TypeVar, cast
+
+from weaviate.cluster.types import Node as NodeREST, Shard as ShardREST
+
+
+@dataclass
+class Shard:
+    """The properties of a single shard of a collection.
+
+    If multi-tenancy is enabled, this corresponds to a single tenant of the collection. Otherwise, this corresponds to the collection itself.
+    """
+
+    collection: str
+    name: str
+    object_count: int
+
+
+@dataclass
+class BatchStats:
```

Comment:
this is only for internal consumption (batching algo), no need to show it to users